src/components/button.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+// this universal button component can be used as a general buttom template to be use across the whole app
+// You do not need to use every prop here for the button to work
+const Button = ({ id, onClick, buttonText, type }) => {
```

Comment:
I have a trick for you that'll make this feel more like a real button. You can remove "buttonText" and replace it with "children". Then when somebody types &lt;Button onclick={() =>{}}&gt;&lt;icon id="arrow" /&gt;Submit&lt;/Button&gt;, both the icon component and the text Submit will come in as the children. Replace {buttonText} below with {children} and you're good to go. 

The best part of this is you can add styling to this button and every button will look identical. Or have a style component that can accept OK buttons vs. Cancel Buttons.

Reply:
@lgtanimoto I only used the button component for the home.js page

Reply:
have now changed the buttonText to children. Thank you Brian for the info.